doc/index-table.md

```diff
@@ -0,0 +1,104 @@
+# Index table
+The `index` type table is used to look up metrics that [match the query](https://graphite.readthedocs.io/en/latest/render_api.html#paths-and-wildcards).
+
+```sql
+CREATE TABLE graphite_index (
+  Date Date,
+  Level UInt32,
+  Path String,
+  Version UInt32
+) ENGINE = ReplacingMergeTree(Version)
+PARTITION BY toYYYYMM(Date)
+ORDER BY (Level, Path, Date);
+```
+
+Where:
+* `Date` - date from received point. Or constant date for full metric list (`1970-02-12` (`toDate(42)`) by default)
+* `Level` - metric name nesting level
+* `Version` - time when the last point was received
```

Comment:
unix timestamp?

Also not really clear why it's needed.

Reply:
Is it better in 9782a7c?